lib/psci/psci_common.c

```diff
@@ -904,7 +904,8 @@ void psci_print_power_domain_map(void)
 				psci_non_cpu_pd_nodes[idx].local_state);
 	}
 
-	for (idx = 0; idx < PLATFORM_CORE_COUNT; idx++) {
+	for (idx = 0; idx < PLATFORM_CORE_COUNT; idx =
+			SPECULATION_SAFE_VALUE(idx + 1)) {
```

Comment:
Again, I'm struggling to see the threat.

Reply:
As discussed offline, we have revised decisions around cold boot path.